### ISSUE TITLE
[VoiceOver] Reader: Cannot exit a Post View Image

### ISSUE THREAD

**Issue by shiki** [wordpress-mobile/WordPress-iOS#12867]
Tap on a post's image. This will show an image preview. Nothing on this page is accessible. It does not seem to be possible to exit.
